apollo/src/graphql/metadata/modelCursor.ts

```diff
@@ -0,0 +1,38 @@
+import { builder } from '../../builder.js';
+import { MLModel } from '../model/model.js';
+import { Model } from 'objection';
+
+export const MLModelCursor = builder.objectRef<ObjectionMLModelCursor>('MLModelCursor');
+
+builder.objectType(MLModelCursor, {
+    fields: (t) => ({
+        cursorId: t.exposeString('cursorId'),
+        cursorToken: t.exposeString('cursorToken'),
+        resultId: t.exposeString('resultId'),
+        expiration: t.exposeString('expiration'),
+    }),
+});
+
+export class ObjectionMLModelCursor extends Model {
+    id!: string;
```

Comment:
Nit: `number`

Reply:
`ObjectionMLModel` is also a string so I kept this a string. I can fix these if you don't mind me fixing them both at once.

Reply:
If you could that'd be sick. Just a lingering brain fart from one of my own prior PRs